merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetActivityTypesAction.java

```diff
@@ -65,7 +66,8 @@ public List<ActivityType> get(final long missionModelId) throws SQLException {
             getJsonColumn(results, "computed_attributes_value_schema", valueSchemaP)
                 .getSuccessOrThrow($ -> new Error("Corrupt activity type computed attribute schema cannot be parsed: "
                                                   + $.reason())),
-            Optional.ofNullable((String) results.getObject("subsystem"))
+            Optional.ofNullable((String) results.getObject("subsystem")),
+            Optional.ofNullable(results.getString("description"))
```

Comment:
Why was `subsystem` using a `.getObject` call?
```suggestion
            Optional.ofNullable(results.getString("subsystem")),
            Optional.ofNullable(results.getString("description"))
```